src/patternfly/components/ModalBox/modal-box-body.hbs

```diff
@@ -1,3 +1,7 @@
-<div class="pf-c-modal-box__body {{ pf-c-modal-box__body--modifier }}">
+<div class="pf-c-modal-box__body{{#if modal-box-body--modifier}} {{modal-box-body--modifier}}{{/if}}"
+  {{#if id}}
+    id="{{id}}"
+  {{/if}}
```

Comment:
I see that `id` is being added to several .hbs files, but I have a few questions about that:
- In the updated examples, I only see `id` being defined for the Close button and the header
- Do we need `id` for the Close button? If we're going to add it, we should use the `btnAttributes` handlebar property that's defined for the Button (see my previous comment about how to format that property value). I don't think `id` is defined as a property for Button, because it isn't showing up in the example html.
- Do we need to include `id` anywhere but `modal-box-header-title.hbs`?
- If we are going to include `id`, should we namespace it like we do the modifier properties (e.g. `modal-box-header-title--id`)?

Reply:
I like this a lot. I've changed the pr to reflect this suggestion. Maybe we want to make ` modal-box-header-title--id` a first class attribute, maybe we want to include it in `component--attributes`. Seems pretty painless to do the latter.

Reply:
In general, I think either method works. The way the modal elements are broken out into separate handlebars, there isn't much benefit in using a specific `--id` property. If all elements were in the same handlebar file, we could use the same handlebar property value for both the `id` and `aria-label` html attribute values. Although maybe there's a way to set that up with the cascade when you have multiple handlebar files? Anyway, I think the way you have it is fine. Thanks!